crates/bindings-csharp/Runtime/LogStopwatch.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpacetimeDB
+{
+    public sealed class LogStopwatch : IDisposable
+    {
+        private uint StopwatchId;
```

Comment:
```suggestion
        private readonly uint StopwatchId;
```

Doesn't have to be mutable as it's supposed to be assigned only once upon construction.

Reply:
Done